cmd/beacon-chain/flags/base.go

```diff
@@ -158,7 +158,7 @@ var (
 	// BlockBatchLimit specifies the requested block batch size.
 	BlockBatchLimit = &cli.IntFlag{
 		Name:  "block-batch-limit",
-		Usage: "The amount of blocks the local peer is bounded to request and respond to in a batch.",
+		Usage: "The amount of blocks the local peer is bounded to request and respond to in a batch. Maximum 128",
```

Comment:
This number changes depending on network so just saying maximum 128 might not be great, we should think of a better way to word this

Reply:
got it. It seems that it's not so good to word this here because the range is dynamic. 
What about adding param judgement on the fetchBlocksFromPeer to make user can realize it without enable the Debug  otherwise user would keep waiting here because there is no any error msg .  like this :
```
maxRequest := params.MaxRequestBlock(slots.ToEpoch(start))
if count > maxRequest {
    log.WithField("count", count).WithField("maxRequest", maxRequest).Error("Requested count is too high, reducing")
}
```